src/Microsoft.TemplateEngine.Abstractions/PublicAPI.Unshipped.txt

```diff
@@ -53,7 +56,12 @@ Microsoft.TemplateEngine.Abstractions.ILocalizationLocator.Identity.get -> strin
 Microsoft.TemplateEngine.Abstractions.ILocalizationLocator.Locale.get -> string!
 Microsoft.TemplateEngine.Abstractions.ILocalizationLocator.Name.get -> string!
 Microsoft.TemplateEngine.Abstractions.ILocalizationLocator.ParameterSymbols.get -> System.Collections.Generic.IReadOnlyDictionary<string!, Microsoft.TemplateEngine.Abstractions.IParameterSymbolLocalizationModel!>!
+Microsoft.TemplateEngine.Abstractions.IParameterSet
+Microsoft.TemplateEngine.Abstractions.IParameterSet.ParameterDefinitions.get -> System.Collections.Generic.IEnumerable<Microsoft.TemplateEngine.Abstractions.ITemplateParameter!>!
 Microsoft.TemplateEngine.Abstractions.IParameterSet.ResolvedValues.get -> System.Collections.Generic.IDictionary<Microsoft.TemplateEngine.Abstractions.ITemplateParameter!, object?>!
+Microsoft.TemplateEngine.Abstractions.IParameterSet.TryGetParameterDefinition(string! name, out Microsoft.TemplateEngine.Abstractions.ITemplateParameter! parameter) -> bool
```

Comment:
Not sure why those haven't been in Shipped/Unshipped files before - there is no change in IParameterSet model...